spring-web-modules/spring-mvc-basics-5/src/main/java/com/baeldung/requestparam/QueryStringController.java

```diff
@@ -0,0 +1,64 @@
+package com.baeldung.requestparam;
+
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+public class QueryStringController {
+
+    @GetMapping("/api/user0")
```

Comment:
I'm wondering, instead of user0/1/2/3/4/5/6 could we instead use a descriptive name instead to make it clearer what is going on without having to look at the implementation? You could use the method names instead, like /api/byGetQueryString

Reply:
Sure